OWNERS_ALIASES

```diff
@@ -24,6 +24,7 @@ aliases:
     - jimangel
     - jlbutler
     - kbhawkey
+    - nate-double-u
```

Comment:
Oh, _also_ it's useful to add a comment that this is for the v1.24 release
I forgot about that until just now.

Reply:
Good point -- I had done that on other permissions requests. Let me open a follow up PR.